packages/document/main-doc/docs/zh/guides/advanced-features/bff/function.mdx

```diff
@@ -141,22 +125,46 @@ export const post = async () => {
 
 - 可以在一个文件中定义多个不同 Method 的函数，但如果定义多个相同 Method 的函数，则只有第一个会生效。
```

Comment:
How about removing this line? With the method function of the same name, the editor will report an error, and this line is useless.

Reply:
OK, i will remove it